### DISCUSSION TITLE
Auf die Konsole im iobroker docker Container zugreifen

### DISCUSSION THREAD

**Discussion by tux-1950** [buanet/ioBroker.docker#493] [Questions & Answers / Fragen & Antworten]
Wir kann ich bei einem laufenden ioBroker docker Container said die console zugreifen? Ich bekommen immer nur eine Fehlermeldung

Error
Unable to retrieve image details

Ich verwende ein Standardimage von Buanet.

**Comment by buanet** [comment]
Auch wenn du es nicht geschrieben hast, wette ich, dass du Portainer benutzt...
Darüber hinaus sollte das auch bei allen anderen Containern passieren. Es gibt/ gab da einen Bug in Portainer im Zusammenhang mit der aktuellen Docker Version. Gib deine Fehlermeldung mal bei Google ein, dann weißt du was ich meine.

Alternativ kannst du jederzeit mit `docker exec` auf die Kommandozeile deiner Container zugreifen. Siehe https://docs.docker.com/reference/cli/docker/container/exec/

MfG,
André

**Reply by tux-1950** [reply]
Vielen Dank für die schnelle Antwort. Richtig, ich nutze Portainer. Das Problem beim Fragestellen ist, dass man oft nicht genau weiß, was alles zur Klärung wichtig ist. 
Werde mal deinen Ratschlag nutzen und es auf diese Art probieren.

**Reply by buanet** [reply]
Ja, schau auch mal nach Portainer. Da ist gerade eine neue Version erschienen, die das Problem löst.

**Reply by tux-1950** [reply]
Habe gerade die neue Version 2.21.0 vom Portainer installiert und der Zugriff auf die Console klappt jetzt. 
Habe es erst mal nur vom Mobiltelefon getestet. Da gibt es noch einen komischen Effekt. Teile der Kommandoeingaben werden bei der Eingabeaufforderung mit ausgegebenen. Muss mal nachsehen, ob das auch am PC so ist 
Vielen Dank für den Hinweis
